student/src/test/java/tds/student/services/remote/RemoteResponseServiceTest.java

```diff
@@ -65,7 +69,17 @@ public void setUp() {
       false,
       mockExamSegmentRepository,
       mockExamItemResponseRepository,
-      mockExamPageRepository);
+      mockExamSegmentWrapperRepository);
+
+    examId = UUID.randomUUID();
+
+    opportunityInstance = new OpportunityInstance(UUID.randomUUID(),
```

Comment:
We weren't using the correct constructor to create teh OpportunityInstance giving it the exam id. Due to that usage we couldn't properly provide mockito with values to get the data.